sootup.jimple.frontend/src/test/java/sootup/jimple/frontend/JimpleStringAnalysisInputLocationTest.java

```diff
@@ -54,15 +60,16 @@ public void test() {
             + "}";
 
     JimpleStringAnalysisInputLocation analysisInputLocation =
-        new JimpleStringAnalysisInputLocation(methodStr);
+        new JimpleStringAnalysisInputLocation(
+            methodStr, SourceType.Application, Collections.emptyList());
 
-    View view = new JimpleView(Collections.singletonList(analysisInputLocation));
-    assertTrue(view.getClass(analysisInputLocation.getClassType()).isPresent());
+    JimpleView view = new JimpleView(Collections.singletonList(analysisInputLocation));
+    assertTrue(view.getClassTypes(analysisInputLocation).isPresent());
 
     MethodSignature methodSig =
         view.getIdentifierFactory()
             .getMethodSignature(
-                analysisInputLocation.getClassType(),
+                view.getClassTypes(analysisInputLocation).get(),
```

Comment:
please remove that method and use:
```suggestion
                view.getIdentifierFactory().getClassType("package.CLASS")
```